### ISSUE TITLE
Block production schedule was removed/gone around 1h before the schedule

### ISSUE THREAD

**Issue by DSNC** [input-output-hk/jormungandr#1884]
**Describe the bug**
After my pool "DSNC" was scheduled for block production, just around 1h before the time the schedule has been removed/cleared and my pool didn't create the block.

**Mandatory Information**
1. `jcli --full-version` output;
jcli v0.8.13 (HEAD-241b3a59, release, linux [x86_64]) - [rustc 1.41.1 (f3e1a954d 2020-02-24)]
2. `jormungandr --full-version` output;
jormungandr v0.8.13 (HEAD-241b3a59, release, linux [x86_64]) - [rustc 1.41.1 (f3e1a954d 2020-02-24)]
**To Reproduce**
Steps to reproduce the behaviour:
1. $ leader_logs
2. it shows my stake pool was scheduled for block production. (see the photo below)
3. $stop (then) $start_leader
4. after the restart my block schedule has gone or cleared out.

**Expected behaviour**
I didn't expect the schedule to get removed or cleared out!
![leader-logs 04-03-2020](https://user-images.githubusercontent.com/37350533/75936672-b7268f00-5e7a-11ea-8bef-fa26fc007693.jpg)
![leader-logs after restart 04-03-2020](https://user-images.githubusercontent.com/37350533/75936692-cd344f80-5e7a-11ea-8785-eff959b0b950.jpg)

**Additional context**

**Comment by rdlrt** [comment]
1. Please use native syntax (not aliases/functions) when reporting errors to this repo for consistency
2. Was the restart _after_ the block was created? If so, that is an expected outcome.

**Comment by DSNC** [comment]
> 1. Please use native syntax (not aliases/functions) when reporting errors to this repo for consistency
> 2. Was the restart _after_ the block was created? If so, that is an expected outcome.

No, my pool was stopped updating the latest block for a long time and I stop the node, clear the logs and start the node again a few hours before the block schedule. 
The node synchronised again, but when I check the leader_logs around an hour before my block producing schedule was gone or removed somehow. (second photo above)
